---
arturl_encode: "68747470733a2f2f62:6c6f672e6373646e2e6e65742f6d305f33373539313737372f:61727469636c652f64657461696c732f313231383537373335"
layout: post
title: "python3执行except-Exception,-e报错"
date: 2024-04-26 20:43:22 +0800
description: "执行代码时，发现报错except TimeoutException,e:原因如下：此代码只在pyth"
keywords: "python3 except exception,e"
categories: ['Python']
tags: ['无标签']
artid: "121857735"
image:
    path: https://api.vvhan.com/api/bing?rand=sj&artid=121857735
    alt: "python3执行except-Exception,-e报错"
render_with_liquid: false
featuredImage: https://bing.ee123.net/img/rand?artid=121857735
featuredImagePreview: https://bing.ee123.net/img/rand?artid=121857735
---

# python3执行except Exception, e报错

![](https://img-home.csdnimg.cn/images/20240711112329.png)

[python](https://so.csdn.net/so/search/s.do?q=python&t=all&o=vip&s=&l=&f=&viparticle=&from_tracking_code=tag_word&from_code=app_blog_art)

执行代码时，发现报错except TimeoutException,e:
  
原因如下：

此代码只在python2.x可使用，python3已不再使用此方法。
  
解决办法：

对应的版本切换到python2
  
将
  
except Exception e: 修改为 except Exception as e: